routers/web/repo/issue_content_history.go

```diff
@@ -68,9 +69,15 @@ func GetContentHistoryList(ctx *context.Context) {
 			actionText = ctx.Locale.Tr("repo.issues.content_history.edited")
 		}
 		timeSinceText := timeutil.TimeSinceUnix(item.EditedUnix, lang)
+
+		username := item.UserName
+		if setting.UI.DefaultShowFullName {
+			username = item.UserFullName
+		}
```

Comment:
Don't you also need to check that the full name is not ""?

```suggestion
		if setting.UI.DefaultShowFullName && strings.TrimSpace(item.UserFullName) != "" {
			username = strings.TrimSpace(item.UserFullName)
		}
```

Reply:
yes, good catch